src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,93 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+  @Column(nullable = false)
+  private String name;
+  @Column(nullable = false)
+  private LocalDate birthDate;
+
+  @Column(nullable = false)
+
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "book_user",
+      joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+      inverseJoinColumns = @JoinColumn(name = "user_id",
+          referencedColumnName = "id"))
+  private List<Book> books;
```

Comment:
You should assign an implementation here
`privateList<Book> books = new ArrayList<>();`

Reply:
Oops

Reply:
I assume declaring no type in the right-hand part makes it auto-detect from the left-hand, but is it more standard to do:
`privateList<Book> books = new ArrayList<>();`
or 
`privateList<Book> books = new ArrayList<Book>();`
?

Reply:
Since java 1.7 it's "legal" not to define it if you're ok with the type on the left hand side.